apps/backend/src/app.service.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  BeforeApplicationShutdown,
+  Injectable,
+  OnApplicationBootstrap,
+  OnApplicationShutdown
+} from '@nestjs/common';
+import os from 'os';
+import winston from 'winston';
+
+@Injectable()
+export class AppService
+  implements
+    OnApplicationBootstrap,
+    BeforeApplicationShutdown,
+    OnApplicationShutdown
+{
+  public logger = winston.createLogger({
+    transports: [new winston.transports.Console()],
+    format: winston.format.combine(
+      winston.format.timestamp({
+        format: 'MMM-DD-YYYY HH:mm:ss Z'
+      }),
+      winston.format.printf((info) => `[${[info.timestamp]}] ${info.message}`)
+    )
+  });
+
+  onApplicationBootstrap(): void {
+    this.logger.info({
+      message: `Started Heimdall Enterprise Server 2.0 on ${os.hostname()} (${os.platform()} ${os.release()}) with PID ${
```

Comment:
If we could remove 2.0 so we don't need to figure out how to keep this updated that would be good.